Need to document required user roles for the BIG-IP controllers.

#### Description
Configured a BIG-IP user with manager permission on dedicated partition, this partition also held the vxlan self, and tunnel.  When instantiated the CC I receive this error:
2017/11/15 01:59:01 [ERROR] [2017-11-15 01:59:01,098 f5_cccl.utils.network ERROR] Failed to configure the FDB for VxLAN tunnel /K8s/osevxlan: 400 Unexpected Error: Ba                       d Request for uri: https://10.105.157.201:443/mgmt/tm/net/fdb/tunnel/~K8s~osevxlan/?ver=11.5.0

If I set the user to admin privileges everything works.  With the manager permissions the VIP and Pool objects are created it looks like that the fdb table entries do not get updated.

#### Kubernetes Version
OCP 3.6

#### Controller Version
1.3

#### BIG-IP Version
12.1.1
#### Diagnostic Information
<Configuration files, error messages, logs>
Note: Sanitize the data. For example, be mindful of IPs, ports, application names and URLs
Note: The following F5 article outlines the information required when opening an issue.
https://support.f5.com/csp/article/K60974137

--- a/docs/README.rst
+++ b/docs/README.rst
@@ -111,6 +111,7 @@ BIG-IP system
 | bigip-url             | string  | Required | n/a               | BIG-IP admin IP address                 |                |
 +-----------------------+---------+----------+-------------------+-----------------------------------------+----------------+
 | bigip-username        | string  | Required | n/a               | BIG-IP iControl REST username           |                |
+|                       |         |          |                   | [#username]_                            |                |
 +-----------------------+---------+----------+-------------------+-----------------------------------------+----------------+
 
 Kubernetes
@@ -133,7 +134,7 @@ Kubernetes
 |                       |         |          |                   | only nodes with this label              |                |
 +-----------------------+---------+----------+-------------------+-----------------------------------------+----------------+
 | pool-member-type      | string  | Optional | nodeport          | The type of BIG-IP pool members you want| cluster,       |
-|                       |         |          |                   | to create.                              | nodeport       |
+|                       |         |          |                   | to create. [#username]_                 | nodeport       |
 |                       |         |          |                   |                                         |                |
 |                       |         |          |                   | Use ``cluster`` to create pool members  |                |
 |                       |         |          |                   | for each of the endpoints for the       |                |
@@ -655,6 +656,7 @@ Example Configuration Files
 .. [#objectpartition] The |kctlr| creates and manages objects in the BIG-IP partition defined in the `F5 resource </containers/latest/kubernetes/index.html#f5-resource-properties>`_ ConfigMap. **It cannot manage objects in the** ``/Common`` **partition**.
 .. [#nodeport] The |kctlr| forwards traffic to the NodePort assigned to the Service by Kubernetes. See the Kubernetes `Services <http://kubernetes.io/docs/user-guide/services/>`_ documentation for more information.
 .. [#secrets] You can `secure your BIG-IP credentials </containers/latest/kubernetes/kctlr-secrets.html#secure-your-BIG-IP-credentials>`_ using a Kubernetes Secret.
+.. [#username] The BIG-IP user account must have an appropriate role defined.  For ``nodeport`` type pool members, this role must be either ``Administrator``, ``Resource Administrator``, or ``Manager``. For ``cluster`` type pool members, the user account must have either the ``Administrator`` or ``Resource Manager`` role. See `BIG-IP Users <https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-concepts-11-5-0/10.html>`_ for further details.
 .. [#hm1] Required if defining the ``virtual-server.f5.com/health`` Ingress/Route annotation.
 .. [#hm2] See the **HTTP monitor settings** section of the `BIG-IP LTM Monitors Reference Guide <https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/bigip-local-traffic-manager-monitors-reference-13-0-0/3.html>`_ for more information about defining send strings.
 

